pretix_eth/network/helpers.py

```diff
@@ -0,0 +1,43 @@
+def make_erc_681_url(
+    to_address, payment_amount, chain_id=1, is_token=False, token_address=None
+):
+    """Make ERC681 URL based on if transferring ETH or a token like DAI and the chain id"""
+    if is_token:
+        if token_address is None:
+            raise ValueError(
+                "if is_token is true, then you must pass contract address of the token."
+            )
+
+        return f"ethereum:{token_address}@{chain_id}/transfer?address={to_address}&uint256={payment_amount}"  # noqa: E501
```

Comment:
I don't think it matters but I have updated anyway. 
I wanted to avoid the extra if condition :)

Reply:
it can actually matter as it is part of a QR code - and the less data is in a QR code the easier it is to scan. Also we have seen some wallets having problems with a chainID here - so not having it when not needed is also an advantage in this way